### PR TITLE
moving cluster startup to what's next and some cleanup

### PR DESCRIPTION
**Pull request by jseldess**
`install-cockroachdb.md` needs to be html, not markdown, for the tabs and styling to work. To make it easy to get correct html, I created `install-cockroachdb-in-md.md`, which you can serve up, view source, and paste the relevant bits of html into `install-cockroachdb.md`. Added a comment to that effect at top of `install-cockroachdb.md`.  

Also moved cluster startup link from steps to a **What's Next** section.

### CODE REVIEW

File: install-cockroachdb-in-md.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+4. Start a new Docker container and load the CockroachDB image into it:
+
+   ~~~ shell
+   $ docker run -t -i cockroachdb/cockroach shell
+   ~~~ 
+
+## Use Docker - Linux
+
+1. [Install Docker](https://docs.docker.com/mac/step_one/).
```
Comment (bdarnell) [inline install-cockroachdb-in-md.md]: This link should point to the linux version instead of the mac version.

File: install-cockroachdb-in-md.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+   $ make build
+   ~~~
+
+    The first time you run `make`, it can take awhile to download and install various dependencies.
+
+## Use Docker - Mac & Windows
+
+<!-- For Windows, remove "Open Launchpad and" from step 2. -->
+
+1. [Install Docker](https://docs.docker.com/mac/step_one/).
```
Comment (bdarnell) [inline install-cockroachdb-in-md.md]: Should this be two links for mac & windows?

**Comment by bdarnell** [comment]
Looks fine to me, but I feel like there must be a better solution to the markdown/html problem.

File: install-cockroachdb-in-md.md (hunk 1 above)
Comment (jseldess) [inline install-cockroachdb-in-md.md]: Good catch. Same is true for windows. Fixing with next commit.

File: install-cockroachdb-in-md.md (hunk 2 above)
Comment (jseldess) [inline install-cockroachdb-in-md.md]: Yeah, adding the windows link to the comment.

**Comment by jseldess** [comment]
I agree about the markdown/html problem. I'll open an issue for post-beta.

**Pull request merged**